router.go

```diff
@@ -942,6 +886,92 @@ func (p *ControllerRegister) handleParamResponse(context *beecontext.Context, ex
 	}
 }
 
+//Invoke the request handler
+func (p *ControllerRegister) invokeRequestHandler(invokeCtx *invokeContext) {
```

Comment:
warning: cyclomatic complexity 22 of function (*ControllerRegister).invokeRequestHandler() is high (> 15) (gocyclo)